config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root 'admins/calculators#index'
```

Comment:
can you make root path as `/calculators` not to admin side. As it said in requirements

Reply:
fixed